assets/src/edit-story/components/library/panes/text/textSets/test/textSet.js

```diff
@@ -275,23 +270,4 @@ describe('TextSets', () => {
       'The possibilities for innovation are not, by any means, exhausted. Technological development is always offering new opportunities for innovative design. But innovative design always develops in tandem with innovative technology, and can never be an end in itself.'
     );
   });
-
-  it('should allow inserting a text set', () => {
```

Comment:
Removing this since we're relying on Moveable for adding a text set and that only displays when hovering over the text set -- can't really test it here.

Reply:
Do we also rely on Moveable for just a click?

Reply:
Yes, previously it was a mix of Moveable vs not Moveable which was not correct really. Basically if was really only Moveable before as well, the non-Moveable was really a backup but it's not needed.

We do this on the element on canvas as well -- Moveable detects a click as dragging and we have to verify inside the Moveable that it's actually a click. If it's a click, we select the element.